gecko_client/xchain_api.go

```diff
@@ -0,0 +1,137 @@
+package gecko_client
+
+import (
+	"encoding/json"
+	"github.com/palantir/stacktrace"
+)
+
+const (
+	xchainEndpoint = "ext/bc/X"
+)
+
+type XChainApi struct {
+	rpcRequester jsonRpcRequester
+}
+
+func (api XChainApi) ImportKey(username string, password string, privateKey string) (string, error) {
```

Comment:
We're adding a bunch of endpoints but no tests to verify that it works as expected. I'm happy to do this; just give me the greenlight and I'll create a branch off this branch, generate a bunch of tests for these guys, and submit a PR against yours

Reply:
Dope added https://github.com/kurtosis-tech/ava-e2e-tests/issues/52 to track